### PR TITLE
remove hardCode

### PR DESCRIPTION
**Pull request by Maki44**
This pull request refactors the globalSettings method in the CommonController to remove hardcoded data and implement dynamic data fetching from the database.

### CODE REVIEW

File: app/Http/Controllers/CommonController.php [hunk 1]
```diff
@@ -216,112 +218,135 @@ public function globalSettings1(Request $request): JsonResponse
...
-        $globalSettings['page_types'] = $pageTypes;
+    $labels = Dictionary::where('language', $language)
+        ->where('tenant_id', $tenantId)
+        ->pluck('value', 'key')
+        ->toArray();
+  
 
-        $globalSettings['content_pages'] = $pageConfig;
+     // Use English labels if requested language labels are not found
+    if (empty($labels)) {
```
Comment (misghna) [inline app/Http/Controllers/CommonController.php]: $labels = Dictionary::where('language', $language)
        ->where('tenant_id', $tenantId)
        ->pluck('value', 'key')
        ->toArray();
  

     // Use English labels if requested language labels are not found
    if (empty($labels)) {
        $labels = Dictionary::where('language', 'english')
            ->where('tenant_id', $tenantId)
            ->pluck('value', 'key')
            ->toArray();
    }
    
    _This implementation is shallow_!  what if only partial labels are in db for that specific language? 
    we need to join it on the fly ...
   
    Use $labels = DB::select(... 
    
    **Hint :**  
    select key, ifnull(x.value,en.value) as value from 
(select key, value from dictionary
where language='english' and tenant_id=123) en
left join 
(select key, value from dictionary
where language='tigrigna' and tenant_id=123) x
ON x.key=y.key

(2) can you remove the for loop and use just an alias 

  $langConfig = [];
    foreach ($langConfigData as $item) {
        $langConfig[] = [
            "id" => $item["lang_id"],
            "name" => $item["lang_name"]
        ];
    }

Comment (Maki44) [reply app/Http/Controllers/CommonController.php]: @misghna For comment number 2, I tried to use alias .
`$langConfig = Language::select('lang_id as id', 'lang_name as name')->get()->toArray();
    $globalSettings['langs'] = $langConfig;` 
    However, using id as it is expected to be the primary key, caused unexpected behavior when used as an alias, returning this:
    `"langs": [
        {
            "id": 0,
            "name": "English"
        },
        {
            "id": 0,
            "name": "ኣምሓርኛ"
        },
        {
            "id": 0,
            "name": "ትግርኛ"
        },
        {
            "id": 0,
            "name": "عربي"
        },
        {
            "id": 0,
            "name": "española"
        }
    ],` As you can see, the id values are zero. That is the reason I used a loop to match the response. Alternatively, I can use Id with a capital letter. What do you think?

Comment (misghna) [reply app/Http/Controllers/CommonController.php]: Hmmmm interesting, may be we can change the it as “value” instead of id!

**Review by misghna — APPROVED** [review]
Thanks, Good work!

**Pull request merged**